src/helpers/analyticsEvents/highlighting/summaryPopup/openClose.ts

```diff
@@ -0,0 +1,24 @@
+import { createSelector } from 'reselect';
+import * as selectNavigation from '../../../../app/navigation/selectors';
+import { AnalyticsEvent } from '../../event';
+
+const open = 'REX highlighting (open summary popup)';
+const close = 'REX highlighting (close summary popup)';
```

Comment:
Didn't check if this was specified in the issue, but in other events, we are using `MH popup`